hex/helpers/lib/check_update.exs

```diff
@@ -74,7 +74,7 @@ case UpdateChecker.run(dependency_name, credentials) do
     version = :erlang.term_to_binary({:ok, version})
     IO.write(:stdio, version)
 
-  {:error, %Version.InvalidRequirementError{} = error}  ->
+  {:error, %Hex.Version.InvalidRequirementError{} = error}  ->
```

Comment:
This will fail on Hex 2.0, which will prove this install method is working as expected. I'll remove it before final merge.